src/main/java/org/folio/dataexp/domain/entity/JobExecutionExportFilesEntity.java

```diff
@@ -35,4 +36,10 @@ public class JobExecutionExportFilesEntity {
 
   @Enumerated(EnumType.STRING)
   private JobExecutionExportFilesStatus status;
+
+  public void setStatusBaseExportStatistic(ExportStrategyStatistic exportStatistic) {
+    if (exportStatistic.getFailed() == 0 && exportStatistic.getExported() > 0) this.status = JobExecutionExportFilesStatus.COMPLETED;
```

Comment:
Please, follow standard convention for if-statements https://www.oracle.com/java/technologies/javase/codeconventions-statements.html.

Reply:
Updated thanks.